packages/plugins/users-permissions/admin/src/pages/Roles/tests/CreatePage.test.js

```diff
@@ -90,7 +90,8 @@ describe('Roles – CreatePage', () => {
     expect(getAllByText('Invalid value')).toHaveLength(2);
   });
 
-  it('can create a new role and show a notification', async () => {
+  // TODO: this test needs to be updated, because it is flakey
+  it.skip('can create a new role and show a notification', async () => {
```

Comment:
Is it having issues in the CI? For me it always passes locally.

Reply:
For me too, but it doesn't in the CI, which makes sense looking at the test setup.

Reply:
I'll start this work here: https://github.com/strapi/strapi/pull/17650 These pages are a bit --- messy.